### DISCUSSION TITLE
希望可以增加仅翻译选中区域的功能

### DISCUSSION THREAD

**Discussion by LingHsvan** [immersive-translate/immersive-translate#1884] [Ideas]
### 新功能 | New Feature

鼠标左键选中部分内容，通过右键菜单或快捷键，“将所选内容转换为{target_language}”
![image](https://github.com/immersive-translate/immersive-translate/assets/91541821/a436fe3a-f37e-45b4-8345-8c76f97691b5)

### 为什么您想要添加这些功能 | Why you want to add these features

翻译较长网页的时候很容易卡死，但又有很多时候并不需要翻译整个网页，而一个一个段落鼠标悬停翻译又很费时间，且有时悬停翻译还没触发，所以需要一个这样的功能

**Comment by KnIfER** [comment]
感觉原插件更流畅？immersive 有时不是实时的，而是一下子翻译了好多
